Guided Investigation - Azure WAF SQLI.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # ToDo add support if not an OWASP rule
</pre><p>The "todo" is done right? in the else clause?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Removed, thanks!